packages/ios_platform_images/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -229,9 +229,12 @@
 			files = (
 			);
 			inputPaths = (
+				"${PODS_ROOT}/Target Support Files/Pods-Runner/Pods-Runner-frameworks.sh",
+				"${BUILT_PRODUCTS_DIR}/ios_platform_images/ios_platform_images.framework",
 			);
```

Comment:
looks like needing a clean.

Reply:
I meant this is not necessary to check in.

Reply:
Why wouldn't we want to apply migrations? It's not *necessary*, since the `flutter` tool will just re-apply them every time it builds, but why not land them?

Reply:
Sure, it's no harm to land it. Let's do it :)